src/components/search-bar/search-bar.jsx

```diff
@@ -37,11 +59,31 @@ export default class SearchBar extends React.Component {
       }
     }
     if (this.props.onSearch) {
-      const query = url ? {url} : null;
-      this.props.onSearch(query);
+      const query = url ? {url} : {url: null};
+      this.urlSearch(query);
     }
```

Comment:
This probably doesn’t need the `if (this.props.onSearch)` guard anymore.

I’m also not sure it makes sense to make `urlSearch` a separate function from the code here, either. Maybe better to just collapse the two together?